<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="sp1_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Spieler1</string>
    </property>
   </widget>
   <widget class="QLabel" name="sp1_karte">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/placeholder.png</pixmap>
    </property>
   </widget>
   <widget class="QSlider" name="sp1_kartenslider">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>21</width>
      <height>150</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sp1_legen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Karte legen</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sp1_uno">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>265</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>UNO</string>
    </property>
   </widget>
   <widget class="QLabel" name="stapel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/placeholder.png</pixmap>
    </property>
   </widget>
   <widget class="QCheckBox" name="sp2_uno">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>265</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>UNO</string>
    </property>
   </widget>
   <widget class="QSlider" name="sp2_kartenslider">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>21</width>
      <height>150</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sp2_karte">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>100</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/placeholder.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="sp2_legen">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Karte legen</string>
    </property>
   </widget>
   <widget class="QLabel" name="sp2_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Spieler2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="neues_spiel_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Neues Spiel</string>
    </property>
   </widget>
   <widget class="QLabel" name="informationen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Informationen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sp1_anzahl_karten">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kartenanzahl:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sp2_anzahl_karten">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kartenanzahl:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="accessibleName">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
